backend/src/models/associationsetup.ts

```diff
@@ -7,33 +7,87 @@ import { database } from "../config/database";
 
```

Comment:
is associationsetup meant to be run as a test/demo?

Reply:
it is run once whenever the app is started, it is where the relationships between models are defined through sequelize and the ORM syncs up with the database.

Reply:
Yes, the relationships are also defined in the column definitions of the models in each particular model class, but for whatever reason sequelize also wants you to define them like this.

Reply:
Sequelize was only helpful up until now lol